src/WaxSigningApi.ts

```diff
@@ -208,15 +237,18 @@ export class WaxSigningApi {
         verified,
         signatures,
         whitelistedContracts,
-        serializedTransaction
+        serializedTransaction,
+        startTime
```

Comment:
should not be receiving this from the back end

Reply:
You should change `receiveSignatures` parameters like so:
`receiveSignatures(startTime: Number, metricName: string,  event: {data: any})`